server/the-usher-openapi-spec.yaml

```diff
@@ -1217,15 +1240,23 @@ components:
     Client:
       type: object
       properties:
+        key:
+          type: integer
+          minimum: 1
+          format: int32
         client_id:
           $ref: '#/components/schemas/EntityNameDef'
         name:
           $ref: '#/components/schemas/EntityNameDef'
         description:
-          $ref: '#/components/schemas/EntityDescriptionDef'
+          type: string
         secret:
           type: string
           maxLength: 50
+        created_at:
+          type: string
```

Comment:
@k1ch so you want to add the format attribute to indicate these are timestamp attributes?